android/sentry.properties

```diff
@@ -0,0 +1,4 @@
+defaults.url=https://sentry.io/
+defaults.org=zooniverse-27
+defaults.project=mobile-app
+auth.token=906b01e443c44171a3fb2a9ef0bddb1d839fe8ce56634d04afe19ad1d5924258
```

Comment:
Should this be hidden? I saw the note about the DSN being in the repo, but having checking this into version control seems strange to me, unless the docs say otherwise.

Reply:
YIKES. Yes, that should be hidden. Let me fix that. Thank you for catching that!